addons/TBMod_rhs/CfgVehicles.hpp

```diff
@@ -24,5 +24,390 @@ class CfgVehicles
+        accuracy = 0.42; // 0.5
+        accuracyDarkNightLightsOff = 0.002; // 0.001
+        accuracyNightLightsOff = 0.012; // 0.006
+        accuracyNightLightsOn = 0.2; // 0.1
+        armor = 55; // 40
+        audible = 64; // 50
+        camouflage = 88; // 100
+        incomingMissileDetectionSystem = 16; // 0
+        irTargetSize = 0.86; // 0.9
+        LockDetectionSystem = "2 + 8 + 4"; // 0
+        magazines[] = {"300Rnd_CMFlare_Chaff_Magazine","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM"}; // "168Rnd_CMFlare_Chaff_Magazine","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM"
+        nvScanner = 1; // 0
+        radarTargetSize = 1.25; // 1
+        weapons[] = {"rhsusf_weap_CMFlareLauncher","rhsusf_weap_ANALQ212"}; // "rhsusf_weap_CMFlareLauncher","rhsusf_weap_ANALQ144"
+    };
+
+    class rhsusf_CH53E_USMC;
+    class rhsusf_CH53E_USMC_D : rhsusf_CH53E_USMC // RHS CH-53
+    {
+        accuracy = 0.3; // 0.5
+        accuracyDarkNightLightsOff = 0.003; // 0.001
+        accuracyNightLightsOff = 0.018; // 0.006
+        accuracyNightLightsOn = 0.3; // 0.1
+        armor = 70; // 40
+        audible = 70; // 50
+        camouflage = 130; // 100
+        incomingMissileDetectionSystem = 16; // 2+8+16
+        irTargetSize = 1.24; // 1.2
+        LockDetectionSystem = "2 + 8 + 4"; // 8
+        magazines[] = {"300Rnd_CMFlare_Chaff_Magazine","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM"}; // "168Rnd_CMFlare_Chaff_Magazine","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM"
+        radarTargetSize = 1.5; // 1.12
+        radarType = 8; // 4
+        weapons[] = {"rhsusf_weap_CMFlareLauncher","rhsusf_weap_ANALQ212"}; // "rhsusf_weap_CMFlareLauncher","rhsusf_weap_ANAAQ24"
+    };
+
+    class RHS_CH_47F_base;
+    class RHS_CH_47F : RHS_CH_47F_base // RHS CH-47
+    {
+        accuracy = 0.35; // 0.5
+        accuracyDarkNightLightsOff = 0.003; // 0.001
+        accuracyNightLightsOff = 0.018; // 0.006
+        accuracyNightLightsOn = 0.3; // 0.1
+        armor = 75; // 30
+        audible = 74; // 50
+        camouflage = 120; // 100
+        irTargetSize = 1.14; // 1.2
+        magazines[] = {"300Rnd_CMFlare_Chaff_Magazine","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM"}; // "168Rnd_CMFlare_Chaff_Magazine","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM","rhsusf_mag_DIRCM"
+        radarTargetSize = 1.65; // 1.12
+        radarType = 8; // 4
+        weapons[] = {"rhsusf_weap_CMFlareLauncher"}; // "rhsusf_weap_CMFlareLauncher","rhsusf_weap_ANALQ212"
```

Comment:
Warum der `rhsusf_weap_ANALQ212` entfernt?

Reply:
Einzig und allein nur, damit alle Helikopter dieses System verwenden, es eine Ausnahme gab, und in Zukunft bei möglichen Änderungen nicht überprüft werden muss welcher Heli welches System hatte. Kann aber auch gerne zurück genommen werden.

Reply:
Grundgedanke war alle Helikopter mit IR-Abwehr auf ein einzelnes System zu bringen, um bei möglichen Änderungen nicht erst überprüfen zu müssen, welcher Heli welches IR-System aktuell verwendet. Es sollten aber keine entfernt werden, die bereits welche besaßen